benchmarking/smartseq2/merge_picard_metrics/merge_picard_metrics.wdl

```diff
@@ -5,14 +5,13 @@ task MergePicardMetrics {
   String output_name
   command {
     set -e
-    gsutil cp  ${target_dir}/merge_picard_mets.py ./   
-    python merge_picard_mets.py  -u ${uuid} -m ${met_name} -o ${output_name}
+    python /usr/local/scripts/merge_picard_mets.py  -u ${uuid} -m ${met_name} -o ${output_name}
```

Comment:
target_dir above isn't used, can be removed

Reply:
Would be preferable to pass in an Array[File] to this task so it can just work with files locally rather than having to download and parse the metadata and then download files from the bucket in Python.

You would need to define that Array[File] in your inputs json file, but you could do that in a couple of ways:

1. Declare the metrics file you need as workflow outputs in the test workflow. When the test workflow finishes, copy and paste the list of metrics files from the workflow metadata into your inputs json file.
2. It might be easier to have one big WDL that runs the test WDL and then runs the benchmarking WDL. That would avoid the problem of needing to copy files or copy/paste lists of files altogether. You mentioned some issues with that approach related to various samples failing, but we could work to make the WDL able to handle that better. (We can set Cromwell server config to have invalidate-bad-cache-results = true to fix one issue.)